lib/crypto/c_src/Makefile.in

```diff
@@ -54,10 +54,16 @@ ifeq ($(TYPE),valgrind)
 TYPEMARKER = .valgrind
 TYPE_FLAGS = $(subst -O3,,$(subst -O2,,$(CFLAGS))) -DVALGRIND
 else
+ifeq ($(TYPE),gprof)
+TYPEMARKER = .gprof
+TYPE_EXTRA_CFLAGS = -DGPROF -pg -DERTS_CAN_INLINE=0 -DERTS_INLINE=
+TYPE_CFLAGS = $(CFLAGS) $(TYPE_EXTRA_CFLAGS)
```

Comment:
TYPE_CFLAGS is not used at all.
Not sure if it should be TYPE_FLAGS or removed.

Reply:
Typo, should be TYPE_FLAGS. Changed it to that.